src/main/java/co/rsk/bitcoinj/script/Script.java

```diff
@@ -682,27 +682,12 @@ public boolean isPayToScriptHash() {
      */
     public boolean isSentToMultiSig() {
         try {
-            MultiSigType multiSigType = this.getRedeemScriptParser().getMultiSigType();
-
-            if (MultiSigType.FLYOVER == multiSigType) {
-                multiSigType = getMultiSigTypeFromInternalRedeemScript();
-            }
-            return multiSigType != MultiSigType.NO_MULTISIG_TYPE;
+            return this.getRedeemScriptParser().getM() > 0;
```

Comment:
Let's add a comment above explaining the rationale of this

Reply:
Done